api/v1/services/TrackingService.js

```diff
@@ -0,0 +1,91 @@
+var CheckitError = require('checkit').Error;
+var _ = require('lodash');
+var _Promise = require('bluebird');
+var ms = require('ms');
+
+var cache = require('../../cache').instance();
+var TrackingItem = require('../models/UniversalTrackingItem');
+var errors = require('../errors');
+var utils = require('../utils');
+
+const TrackingFooter = "UniversalTracking";
+
+module.exports.createTrackingEvent = function (attributes) {
+    var trackingItem = TrackingItem.forge(attributes);
+
+    return trackingItem
+        .validate()
+        .catch(CheckitError, utils.errors.handleValidationError)
+        .then(function (validated) {
+            return TrackingItem.findByName(attributes.name);
+        })
+        .then(function (result) {
+            if (!_.isNull(result)) {
+                var message = "This event is already being tracked";
+                var source = "name";
+                throw new errors.InvalidParameterError(message, source);
+            }
+
+            return cache.getAsync('trackedEvent')
+        })
+        .then(function (result) {
+            if(!_.isNil(result)) {
+                return cache.ttlAsync('trackedEvent')
+                    .then(function (ttl) {
+                        var message = result + " is already being tracked at this time, time left: " + ttl.toHMS();
```

Comment:
@redsn0w422 wanted the current tracked event in the message, maybe: `"An event is already being tracked. The current event, " + result + " ends in " + ttl.toHMS();`?